README.md

```diff
@@ -15,10 +15,12 @@ also an IRC channel on OFTC (irc.oftc.net) #gtest available.  Please
 join us!
 
 **Google Mock** is an extension to Google Test for writing and using C++ mock
-classes.  See the separate [Google Mock documentation](googlemock/README.md).
+classes. 
 
-More detailed documentation for googletest (including build instructions) are
-in its interior [googletest/README.md](googletest/README.md) file.
+## Documentation ## 
```

Comment:
Do you mean  -with trailing space visible- ? 
As for ## : 
Cannot reproduce pasting to http://dillinger.io/
I've written ## Name ## to form a new subheader in convenience with ## Features ## , ## Platforms ## . Am I wrong?